src/server.c

```diff
@@ -651,7 +651,7 @@ static bool handle_connect(PgSocket *server)
 	char buf[PGADDR_BUF + 32];
 	bool is_unix = pga_is_unix(&server->remote_addr);
 
-	fill_local_addr(server, sbuf_socket(&server->sbuf), is_unix);
+	fill_local_addr(server, sbuf_socket(&server->sbuf), is_unix, 0);
```

Comment:
Why are we passing 0 here? Would be good to add a comment on `fill_local_addr` and `fill_remote_addr` in which cases 0 is expected.